docs/specs/discovery.md

```diff
@@ -34,18 +36,21 @@ sequenceDiagram
   %%{wrap}%%
   participant Devices
   participant Spotter
-  participant Agent as Agent<br/>(w/ Mapping)
+  participant Registry
+  participant Provisioning Engine
+  participant Mapping Agent
   participant Pipeline
-  Note over Devices, Agent: Discovery Start
-  activate Agent
-  Agent->>Spotter: DISCOVERY CONFIG<br/>()
+  Note over Devices, Provisioning Engine: Discovery Start
+  activate Provisioning Engine
+  Mapping Agent->>Registry: DISCOVERY CONFIG<br/>for Spotter<br/>()
```

Comment:
I'd take out the "for Spotter" and stray () -- they don't add much and then begs the question why every other message doesn't have the same.